core-web/libs/edit-content/src/lib/feature/edit-content/edit-content.layout.component.spec.ts

```diff
@@ -1,23 +1,310 @@
+        expect(spectator.component.identifier).toEqual('1');
     });
 
-    it('should create', () => {
-        expect(component).toBeTruthy();
+    describe('Data from form', () => {
+        it('should call getContentById and getContentTypeFormData with contentType if identifier is present', () => {
+            // TODO: Ask why this dont work
+            // // dotEditContentService.getContentById.mockReturnValue(of(CONTENT_TYPE_MOCK));
+            // const httpService = spectator.inject(HttpClient);
+            // httpService.get.mockReturnValue(of({ entity: CONTENT_TYPE_MOCK }));
+            // dotEditContentService.getContentTypeFormData.mockReturnValue(of(LAYOUT_MOCK));
+
+            // TODO: Ask why this dont work as a spy mock.
+
+            // dotContentTypeService.getContentType.mockReturnValue(of(CONTENT_TYPE_MOCK));
+            spectator.detectChanges();
+
+            expect(dotEditContentService.getContentTypeFormData).toHaveBeenCalledWith('test');
+            // expect(dotEditContentService.getContentTypeFormData).toHaveBeenCalledWith('test');
+        });
+
+        // it('should call getContentTypeFormData with contentType if identifier is not present', () => {
+        //     // const createComponentWithoutId = createComponentFactory({
+        //     //     component: EditContentLayoutComponent,
+        //     //     imports: [CommonModule, DotEditContentFormComponent],
+        //     //     providers: EDIT_CONTENT_LAYOUT_PROVIDERS_MOCK
+        //     // });
+        //     // spectator = createComponentWithoutId();
+        //     spectator.component.identifier = undefined;
+        //     spectator.detectChanges();
+        //     // dotEditContentService = spectator.inject(DotEditContentService);
+        //     jest.spyOn(dotEditContentService, 'getContentTypeFormData');
+        //     dotEditContentService;
+        //     expect(dotEditContentService.getContentTypeFormData).toHaveBeenCalledWith('test');
+        // });
     });
+
+    // describe('saveContent', () => {
+    //     it('should call dotEditContentService.saveContentlet with the correct parameters', () => {
+    //         jest.spyOn(dotEditContentService, 'saveContentlet')
+    //         spectator.component.saveContent({ key: 'value' });
+    //         expect(dotEditContentService.saveContentlet).toHaveBeenCalledWith({
+    //             key: 'value',
+    //             inode: '1',
+    //             contentType: 'test'
+    //         });
+    //     });
+
+    //     it('should set isContentSaved to true and then false after 3 seconds', fakeAsync(() => {
+    //         jest.spyOn(dotEditContentService, 'saveContentlet').and.returnValue(of({}));
+    //         spectator.component.saveContent({ key: 'value' });
+    //         expect(spectator.component.isContentSaved).toBe(true);
+    //         tick(3000);
+    //         expect(spectator.component.isContentSaved).toBe(false);
+    //     }));
+    // });
```

Comment:
It's safe to push broken test if you are working on it. Anyways, if these tests are necessary uncomment them